unity-renderer/Assets/DCLPlugins/BuilderInWorld/HUD/ProjectsPanelHUD/Scripts/NewProject/NewProjectSecondStepView.cs

```diff
@@ -40,7 +40,10 @@ public override void Start()
             base.Start();
 
             rowsInputField.OnInputChange += RowsChanged;
+            rowsInputField.OnInputLostFocus += RowsInputLostFocus;
```

Comment:
Only to let you know... if you had created the `LimitInputField` class as a UI Component (inheriting from `BaseComponentView`), you would have already available the `isFocused` flag with its respectives `OnFocus()` and `OnLoseFocus()` functions that would let you do the same flow you are implemting here.